tf_encrypted/serving/queues.py

```diff
@@ -101,6 +101,8 @@ def __init__(
     self.output1 = output.share1
 
     # fixedpoint config
+    # TODO[jvmncs]: how do we access fixedpoint_config without calling
```

Comment:
ideally the queue should not know about this; afair it is only stored here so we can properly rewrap tensors in `receive_output` etc. this could properly be done by storing some type information instead. don't have to do this now.